src/main/java/codeu/controller/RocketBotHandler.java

```diff
@@ -0,0 +1,55 @@
+package codeu.controller;
+
+import codeu.model.data.BotRequest;
+import codeu.model.data.Message;
+import codeu.model.data.User;
+import codeu.model.store.basic.MessageStore;
+import codeu.model.store.basic.UserStore;
+
+import java.time.Instant;
+import java.util.UUID;
+
+/**
+ * Handles the Rocket Bot. This bot is triggered by the word "rocket" and responds with a
+ * hard-coded message: "3...2...1...BLASTOFF!". The handler first creates a User for the
+ * bot, then handles its actions as outlined in BotHandlerInterface.
+ */
+public class RocketBotHandler implements BotHandlerInterface {
+
+  /** Handles bot requests and sends as messages to conversations. */
+  public void handler(BotRequest request) {
+    final String BOT_NAME = "RocketBot";
+    final String BOT_OUTPUT = "3...2...1...BLASTOFF!";
+    final UUID BOT_USER_ID = UUID.randomUUID();
+
+    // Create User and add the User to UserStore only if it does not already exist.
+    if (userStore.getUserWithId(BOT_USER_ID) == null) {
```

Comment:
`BOT_USER_ID` would change every time this is called, also, so we need a way to store in DataStore that it's a bot specifically for reuse here